pkgs/desktops/gnome-3/extensions/gsconnect/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub
+, meson, ninja, libxml2, pkgconfig
+, glib, folks, nautilus, libcanberra, sshfs }:
+
+stdenv.mkDerivation rec {
+  name = "gnome-shell-gsconnect-${version}";
+  version = "13-alpha";
+
+  src = fetchFromGitHub {
+    owner = "andyholmes";
+    repo = "gnome-shell-extension-gsconnect";
+    rev = "v${version}";
+    sha256 = "0dynq5b2vrp54r9k6fpc81qjmr2qv1nbbyn4khrjxv283q7w3h7q";
+  };
+
+  # TODO: Missing "dbus-1"
+  nativeBuildInputs = [
+    meson ninja pkgconfig
+    libxml2 # xmllint
+  ];
+
+  buildInputs = [
+    glib # libgobject
+    folks # libfolks
+    nautilus
+    # TODO: Look for "nautilus-python"
+    libcanberra
+    sshfs
+  ];
+
+  postPatch = "patchShebangs /build/source/meson/nmh.sh";
+
+  meta = with stdenv.lib; {
+    description = "KDE Connect implementation for Gnome Shell";
+    license = licenses.gpl2;
+    maintainers = with maintainers; [ etu ];
+    homepage = https://extensions.gnome.org/extension/1319/gsconnect/;
```

Comment:
The extension page points to https://github.com/andyholmes/gnome-shell-extension-gsconnect/wiki as a homepage.